<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="sunflower" failures="1" tests="1" name="runner.TestNGTestRunner" time="22.161" errors="0" timestamp="2024-12-18T11:54:54 EST" skipped="0">
  <testcase classname="runner.TestNGTestRunner" name="runScenario" time="22.161">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefinitions.StepDefinitionImplementation.checkout_submit_order(StepDefinitionImplementation.java:47)
at âœ½.Checkout "ZARA COAT 3" and submit the order(file:///C:/eclipse-workspace/SeleniumFrameDesign/src/test/java/features/PurchaseOrder.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runner.TestNGTestRunner -->
